src/main/java/ecdar/presentations/TagPresentation.java

```diff
@@ -40,7 +37,7 @@ public class TagPresentation extends StackPane {
     LineTo l3;
     boolean hadInitialFocus = false;
 
-    static double TAG_HEIGHT = 1.6 * GRID_SIZE;
+    static double TAG_HEIGHT = 16;
```

Comment:
Consider making this a function of `Ecdar.CANVAS_PADDING`

Reply:
IMO unexpected dependency